connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -163,6 +164,14 @@ message Sort {
   }
 }
 
+// Relation of type [[Deduplicate]] which have duplicate rows removed, could consider either only
+// the subset of columns or all the columns.
+message Deduplicate {
+  Relation input = 1;
+  repeated string deduplicate_columns = 2;
```

Comment:
no biggie but I would match this name to what we have in `Dataset.scala`. e.g., col_names